public/xml/herodotus-8-78-83.xml

```diff
@@ -199,7 +199,7 @@
              span="">
       <word id="1" form="ἡμέας" lemma="ἐγώ" postag="p1p---ma-" relation="OBJ" gloss="" sg="" head="2"/>
       <word id="2" form="στασιάζειν" lemma="στασιάζω" postag="v--pna---" relation="SBJ" gloss="" sg="" head="4"/>
-      <word id="3" form="χρεόν" lemma="χρεών" postag="v-spp_nn_" relation="PNOM" gloss="" sg="" head="4"/>
+      <word id="3" form="χρεόν" lemma="χρεών" postag="x-----n--" relation="PNOM" gloss="" sg="" head="4"/>
```

Comment:
This looks correct to me, but I don't think it should be mixed in with the other changes since it's not related to postags that are too long. Maybe a separate PR?